types/telegram-web-app/index.d.ts

```diff
@@ -0,0 +1,306 @@
+// Type definitions for non-npm package telegram-web-app 1.0
+// Project: https://telegram.org/js/telegram-web-app.js
+// Definitions by: KnorpelSenf <https://github.com/KnorpelSenf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare global {
+  interface Window {
+    Telegram: Telegram;
+  }
+}
```

Comment:
```suggestion

declare var Telegram: Telegram;
```
and remove every `export ` keyword. This should support both global script usage and usage via `window.Telegram` (`export` is not needed, as this is global script, not a module)